packages/valory/connections/abci/tests/test_abci.py

```diff
@@ -327,8 +339,19 @@ def make_app(self) -> ABCIAppTest:
         return ABCIAppTest(self.TARGET_SKILL_ID)  # type: ignore
 
 
+def start_tendermint_docker_image() -> None:
+    """Start TendermintDockerImage"""
+
+    client = docker.from_env()
+    image = TendermintDockerImage(client)  # abci_host, abci_port, tendermint_port)
+    container = image.create()
+    container.start()
+    logging.info(f"Setting up image {image.image}...")
+    success = image.wait()
+    logging.info(f"TendermintDockerImage running: {success}...")
+
+
 @pytest.mark.integration
-@pytest.mark.usefixtures("tendermint")
 class BaseTestABCITendermintIntegration(BaseThreadedAsyncLoop, ABC):
```

Comment:
We cannot use the Tendermint autouse fixture, because gRPC only sends a single echo request. Hence, the connection needs to be ready to receive it when the docker image is launched.